public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.test.ts

```diff
@@ -889,3 +898,182 @@ function createAnnotationLayers() {
     }),
   ];
 }
+
+describe('validateDashboardSchemaV2', () => {
+  const validDashboard = {
+    title: 'Test Dashboard',
+    timeSettings: {
+      from: 'now-1h',
+      to: 'now',
+      autoRefresh: '5s',
+      hideTimepicker: false,
+      timezone: 'UTC',
+      autoRefreshIntervals: ['5s', '10s', '30s'],
+      quickRanges: [],
```

Comment:
Isn't the k8s API going to remove this (similarly to `mappings`) ?

Reply:
It's still a valid dashboard :) This is testing that the check the output from the transform is correct, but then, the serializer will clean up the part that should not go to the backend. As it does with null values for the JSON for example :)

Reply:
Yeah, definitelly, I was just wantering if this won't create an unnecessary diff and activate the save button after a dashboard has been saved.